prow/config/config.go

```diff
@@ -770,9 +780,12 @@ func defaultDecorationMapToSlice(m map[string]*prowapi.DecorationConfig) []*Defa
 	var entries []*DefaultDecorationConfigEntry
 	add := func(repo string, dc *prowapi.DecorationConfig) {
 		entries = append(entries, &DefaultDecorationConfigEntry{
-			OrgRepo: repo,
-			Cluster: "",
-			Config:  dc,
+			OrgRepo:               repo,
+			Cluster:               "",
+			Config:                dc,
+			PodPendingTimeout:     nil,
```

Comment:
`nil` is the default value, you do not need to specify it

Reply:
Removed 👍